Tests/RMCInternalSteps.fcl

```diff
@@ -0,0 +1,12 @@
+#include "Production/JobConfig/primary/RMCInternal.fcl"
+source.firstRun: 501 #TODO
+physics.filters.TargetStopResampler.fileNames : [
+"/pnfs/mu2e/persistent/datasets/phy-sim/sim/mu2e/MuminusStopsCat/MDC2020p/art/6d/cf/sim.mu2e.MuminusStopsCat.MDC2020p.001201_00000000.art"]
```

Comment:
I appreciate having test scripts, but there is a lot of replication. When the stops file evolves (which it will) this will require a lot of maintenance to update. Can the repeated parts be consolidated?

Reply:
I moved the muon stop file info (and the run number) to a test fcl file `Production/Tests/MuonStopConfig.fcl` so the muon stop file can more easily be updated. I also updated the CE, DIO, and flat e+/e- test fcl to include this config file to reduce maintenance.